readme.md

```diff
@@ -146,6 +179,30 @@ def destroy
 end
 ```
 
-## Bonus!
+## You do: Prevent Users from editing someone else's post.
+
+## You do: Associate users with Comments
+
+1. Create a new migration to add a user_id column to comments.
+2. Associate the `current_user` with newly created posts.
+3. Prevent User's from editing / deleting other's comments.
+
+## Customizing Views
+
+Devise has generated several views for us, but they are not currently visible in our
+codebase.
+
+If you want to customize them, generate the views with `rails g devise:views`
+
+The possibilities are endless!
+
+### Styling views
+
+Whether you generated the views or not, you can style the forms the same way.
+
+Identify the selectors you'd use to target the individual form elements, and add
+styles in `app/assets/stylesheets/application.css`
+
+## If there's time...
 
-If you need more fine-grained control over user permissions, check out [CanCanCan](https://github.com/CanCanCommunity/cancancan)
+Try implementing an authorization solution on top of devise - https://github.com/ga-wdi-lessons/cancancan
```

Comment:
potential bonus material could also be customized user model attributes, or `rails g devise:controllers`